07/reshape-a-matrix.checkpoint.md

```diff
@@ -250,7 +333,7 @@ def reshape_matrix(matrix, r, c):
 
 ```py
 import unittest
-import main as p
+from main import *
```

Comment:
is all we need reshape_matrix? * is ok, but generally recommended to be avoided. not a big deal since there probably won't be any other stray symbols to import, but something to consider.
```suggestion
from main import reshape_matrix
```